makelib/golang.mk

```diff
@@ -51,7 +51,7 @@ GO_LDFLAGS += -s -w
 endif
 
 # supported go versions
-GO_SUPPORTED_VERSIONS ?= 1.7|1.8|1.9|1.10|1.11|1.12|1.13|1.14
+GO_SUPPORTED_VERSIONS ?= 1.7|1.8|1.9|1.10|1.11|1.12|1.13|1.14|1.15
```

Comment:
Should we update to actually build with go 1.15 also? I think the purpose of this list is to essentially set a maximum supported version. So we usually keep the maximum version in lockstep with the version that we build with so that consumers of the submodule are not surprised by something building locally but not in CI. And in that case, the build submodule bump is needed across repos anyway to actually build with the new version we download in the cross container.